python/ray/train/v2/tests/test_worker_group.py

```diff
@@ -78,6 +78,25 @@ def test_worker_group_create():
         worker_group.get_workers()
 
 
+def test_worker_group_create_with_runtime_env():
+    """Test WorkerGroup.create() factory method with a custom runtime environment."""
+    train_run_context = TrainRunContext(
+        run_config=RunConfig(worker_runtime_env={"env_vars": {"DUMMY_VAR": "abcd"}})
```

Comment:
Should we also test that `get_env_vars_to_propagate` are included?

Reply:
tested here already: https://github.com/ray-project/ray/blob/a9088662c05ef1ba0dd742937bdef7302ad0f946/python/ray/train/v2/tests/test_worker_group.py#L398